hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsConfiguration.java

```diff
@@ -125,6 +125,11 @@
       DefaultValue = MAX_CONCURRENT_WRITE_THREADS)
   private int maxConcurrentWriteThreads;
 
+  @IntegerConfigurationValidatorAnnotation(ConfigurationKey = AZURE_LIST_MAX_RESULTS,
+      MinValue = 1,
+      DefaultValue = DEFAULT_AZURE_LIST_MAX_RESULTS)
```

Comment:
Do we need a MaxValue here?

Reply:
Currently server has a maximum limit which is 5000. values >= 5000 will be defaulted to 5000. Did not want to keep max value 5000 here, in case server in future plans making changes to this.